packages/bun-release/scripts/upload-npm.ts

```diff
@@ -265,7 +272,10 @@ async function test() {
     });
 
     console.log("Testing", install + " bun");
-    await $`${{ raw: install }} ./bun-${version}.tgz`;
+    const nodePath = Bun.which("node")?.replace("/node", "") || "";
+    await $`${{ raw: install }} ./bun-${version}.tgz`.env({
+      PATH: nodePath ? `${nodePath}:${process.env.PATH}` : process.env.PATH,
+    });
```

Comment:
I have no idea why it throws `env: node: No such file or directory` without this force path change...